src/azure-cli/HISTORY.rst

```diff
@@ -6,6 +6,11 @@ Release History
 2.3.1
 ++++++
 
+**SQL**
```

Comment:
This part should be removed and add it in description.

Reply:
No need to change history.rst because we will generate it automatically.

Reply:
Oh okay, where is the description section?

Reply:
https://github.com/Azure/azure-cli/pull/11721#issue-358247353 or title